ports/rp2/modrp2.c

```diff
@@ -65,7 +65,12 @@ static bool __no_inline_not_in_flash_func(bootsel_button)(void) {
 
     // The HI GPIO registers in SIO can observe and control the 6 QSPI pins.
     // The button pulls the QSPI_SS pin *low* when pressed.
-    bool button_state = !(sio_hw->gpio_hi_in & (1 << CS_PIN_INDEX));
+    #if PICO_RP2040
+        #define CS_BIT (1u << 1)
+    #else
+        #define CS_BIT SIO_GPIO_HI_IN_QSPI_CSN_BITS
+    #endif
```

Comment:
I suggest moving this #if block to just above this bootsel_button function (that's the general coding style, not to have #defines within code itself).

Reply:
Good shout, agreed!